wordpress-mobile/WordPress-iOS
# Blaze: Present blaze flow from entry points

Part of #20090 
Part of #20091 
Part of #20092 

- Presents blaze from the menu list, dashboard card, and post list entry points
- Analytics will be adressed in a future PR

## How to test
0. Make sure that Blaze is enabled via the debug menu
1. Switch to a site that has Blaze enabled - let's call this site A (See peeHDf-zm-p2 for more info on Blaze requirements)
2. Switch to the dashboard
3. Tap on the Blaze dashboard card
4. ✅ The Blaze flow is presented in a webview
5. Switch to the site menu
6. Tap on the Blaze menu item
7. ✅ The Blaze flow is presented in a webview
8. Tap on the Posts menu item
9. On a Blaze enabled post, tap on the ellipsis button
10. Tap on "Promote with Blaze"
11. ✅ The Blaze flow is presented in a webview with the post details filled out


Uploading Simulator Screen Recording - iPhone 14 Pro - 2023-02-22 at 15.44.02.mp4…



## Regression Notes
1. Potential unintended areas of impact
Blaze dashboard card

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the "Hide this" option

12. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
